curl/CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ if (BUILDING_LEATHERMAN)
 endif()
 
 add_leatherman_library(src/client.cc src/request.cc src/response.cc)
+if (WIN32)
+    symbol_exports(leatherman_curl "${CMAKE_CURRENT_LIST_DIR}/inc/leatherman/curl/export.h")
```

Comment:
Is this something that should be folded into the `add_leatherman_library` macro? Something like `add_leatherman_library(src1 src2 src3 EXPORTS /path/to/export.h`?